Add a link to the Custom metrics page from the Serverless main page (#17433)
DataDog/documentation: content/en/serverless/_index.md

[Enhanced Lambda metrics][3], which appear in Datadog with the prefix `aws.lambda.enhanced`, are available at second granularity and in near real time. You can use enhanced Lambda metrics for alerts or SLOs on cold starts, estimated AWS costs, timeouts, out-of-memory errors, and memory usage across all of your Lambda functions.

> Send [Custom metrics][4] from a Lambda function through several methods such as generating metrics from logs and traces, the Datadog Lambda Extension, and the Datadog Forward Lambda.

Reviewer: ```suggestion
You can send [custom metrics][4] from a Lambda function by generating metrics from logs and traces, using the Datadog Lambda Extension, or using the Datadog Forwarder Lambda.
```
